Library/Formula/dcadec.rb

```diff
@@ -0,0 +1,16 @@
+class Dcadec < Formula
+  homepage "https://github.com/foo86/dcadec"
+  version "7a5479454156c7a2223c7452f2a965389dce5e67"
+  url "https://github.com/foo86/dcadec/archive/7a5479454156c7a2223c7452f2a965389dce5e67.tar.gz"
```

Comment:
Sorry, we need a tagged stable release to accept this software. Could you maybe request one from the author?